languages/wikipron/src/generate_summary.py

```diff
@@ -0,0 +1,85 @@
+import os
+import json
+import csv
+
+from codes import LANGUAGES_PATH, README_PATH, LANGUAGES_SUMMARY_PATH
+
+
+def main():
+    with open(LANGUAGES_PATH, "r") as source:
+        languages = json.load(source)
+    readme_list = []
+    languages_summary_list = []
+    path = "../tsv"
+    for file_path in os.listdir(path):
+        # Filter out README.md.
+        if file_path.endswith(".md"):
+            continue
+        with open(f"{path}/{file_path}", "r") as tsv:
+            num_of_entries = sum(1 for line in tsv)
+        iso639_code = file_path[: file_path.index("_")]
+        transcription_level = file_path[
+            file_path.rindex("_") + 1 : file_path.index(".")
+        ].capitalize()
+        wiki_name = languages[iso639_code]["wiktionary_name"]
+        if "dialect" in languages[iso639_code]:
+            dialect_key = file_path[
+                file_path.index("_") + 1 : file_path.rindex("_")
+            ]
+            dialects = languages[iso639_code]["dialect"][dialect_key]
+            if "|" in dialects:
+                dialects = dialects.replace(" |", ",")
+            wiki_name += f" ({dialects})"
+
+        row = [
+            iso639_code,
+            languages[iso639_code]["iso639_name"],
+            wiki_name,
+            str(languages[iso639_code]["casefold"]),
+            transcription_level,
+            str(num_of_entries),
+        ]
+        # TSV and README have different first column.
+        languages_summary_list.append([file_path] + row)
+        readme_list.append([f"[TSV]({file_path})"] + row)
+
+    # Sort by wiktionary language name,
+    # with phonemic entries before phonetic.
+    languages_summary_list.sort(key=lambda ele: ele[3] + ele[5])
```

Comment:
Okay, so now you're using this twice, so probably define it outside of main as `_3_and_5` or something like that?